### DISCUSSION TITLE
Map Levels

### DISCUSSION THREAD

**Discussion by reujoal** [JasonLautzenheiser/trizbort#558] [Ideas]
I think it would be very helpful to be able to to change levels. So I could build an "UP/DOWN" exit on the ground levels and connect it to a location that is on the level above it.

**Comment by JasonLautzenheiser** [comment]
So you can make a connection up/down.  Right now it just acts like any other connection (display connection on map, handle export to code properly, ...)

Is there something else visually or functionally you'd like to see?  I have some ideas, but want to hear your thoughts untainted from my own.

Thanks

**Reply by reujoal** [reply]
I'm not sure if you are familiar with Zmud, but its mapping system had the ability to move on the Z axis.

Say "level 1" is the main ground map, Basically, if you wanted to create an 'up' location it would create a blank map on what would be level 2  with a slightly transparent overlay so you could see the map below and match their positions. Along with this, as shown in the top right corner of the screenshot, where was a button to go up and down levels. More than anything it would be an interface feature and would still function the same  when exporting etc.

I hope I explained that properly!
![Creatin2](https://user-images.githubusercontent.com/69661631/163680361-afbdb43b-f6cc-4080-bf32-7a082df084ec.jpg)

**Reply by JasonLautzenheiser** [reply]
I am not familar with zmud, but I like the idea of levels how it utilizes them.   I will look at adding that feature soon.

**Reply by llzzies** [reply]
Love this idea (and have also wished for something like this myself)! Look forward to seeing what you come up with.
